geomesa-gt/geomesa-gt-partitioning/src/test/scala/org/locationtech/geomesa/gt/partition/postgis/PartitionedPostgisDataStoreTest.scala

```diff
@@ -47,7 +47,7 @@ class PartitionedPostgisDataStoreTest extends Specification with BeforeAfterAll
 
   val hours = 1
   val spec =
-    "name:List[String],props:String:json=true,age:Int,dtg:Date,*geom:Point:srid=4326;" +
+    "name:List[String],props:String:json=true,age:Int,dtg_abcdefghijklmnopqrstuvwxyzabcde:Date,*geom_abcdefghijklmnopqrstuvwxyzabcde:Point:srid=4326;" +
```

Comment:
could we revert this change? we could add another test with long attribute names, but since it passes we could maybe skip it. I think the failure case would be something like `reallylongcommonprefix__________dtg:Date,reallylongcommonprefix__________geom:Point` which is maybe not something we need to worry about.